src/compiler/types.ts

```diff
@@ -8259,9 +8267,11 @@ export const enum LanguageFeatureMinimumTarget {
     TopLevelAwait = ScriptTarget.ES2022,
     ClassFields = ScriptTarget.ES2022,
     PrivateNamesAndClassStaticBlocks = ScriptTarget.ES2022, // `class C { static {} #x = y, #m() {} }`, `#x in y`
+    RegularExpressionFlagsHasIndices = ScriptTarget.ES2022,
 
     // ES2023 Features
     ShebangComments = ScriptTarget.ESNext,
+    RegularExpressionFlagsUnicodeSets = ScriptTarget.ESNext,

```

Comment:
I _think_ this is 2023 but I can't find the page where I read that.

Reply:
This is ES2024! See https://tc39.es/ecma262/multipage/#sec-intro and https://github.com/microsoft/TypeScript/pull/57111